afdko/Tools/SharedData/FDKScripts/autohint.py

```diff
@@ -13,7 +15,7 @@
 Auto-hinting program for PostScript and OpenType/CFF fonts.
 """
 
-from ufoTools import kProcessedGlyphsLayerName, kProcessedGlyphsLayer
+from .ufoTools import kProcessedGlyphsLayerName, kProcessedGlyphsLayer
```

Comment:
careful with these relative imports. if the python module is meant to be run as a script, then you can't use relative imports.

Reply:
if one attempts to run a python script containing a relative import, one gets
`ValueError: Attempted relative import in non-package` (py2) or `ModuleNotFoundError:... '__main__' is not a package` (py3)

Reply:
will revert

Reply:
note that the issue with scripts and relative imports only arises if one attempts to call the module as a script like in `python afdko/Tools/SharedData/FDKScripts/autohint.py`.
If the the module has a setuptools controle_scripts entry point defined in setup.py, then it will be imported as a module and it's main() will be called from the wrapper script. So in this case, the name of the module is not `__main__` but the full qualified module name.

Reply:
reverted ac1494d1c3ba2a0fd99b13634b8aae217552ba92